widgetastic_manageiq/__init__.py

```diff
@@ -1261,7 +1261,17 @@ def items_per_page(self):
         return int(re.sub(r'\s+items', '', selected))
 
     def set_items_per_page(self, value):
-        self.items_on_page.select_by_visible_text(str(value))
+        """Selects number of items to be displayed on page.
+
+        Args:
+            value: Ideally a str of the format 'x items', x could be {10,20,50,..,1000}
+                   but if value is a number, 'items' is added to it as suffix
+        """
+        if not isinstance(value, (int, six.string_types)):
+            raise TypeError("Value should either be of format either e.g. 10 or a string " +
```

Comment:
Don't use `+` to concatenate these strings here, the line continuation will work just fine without it